### ISSUE TITLE
EA suggests to declare :void return type to a method that returns null

### ISSUE THREAD

**Issue by 5lava** [kalessil/phpinspectionsea#1627]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 4.0.5 |
| Language level | PHP 7.4 |

Current behaviour
--
```php
new class {
    public function ouch() { return null; }
};
```

![image](https://user-images.githubusercontent.com/63829/92133839-d9bc4500-edd6-11ea-87fa-2808044ab60c.png)

Expected behaviour
--
No inspection warning.

Environment details
---
PhpStorm 2020.2.1
Build #PS-202.6948.87, built on August 27, 2020
Runtime version: 11.0.8+10-b944.31 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8
Registry: run.processes.with.pty=TRUE, undo.documentUndoLimit=10000, ide.allow.merge.buttons=false, ide.tooltip.initialDelay=0, php.brace.alt.syntax=true
Non-Bundled Plugins: PHPUnit code coverage, org.intellij.plugins.postcss, ideanginx9, name.kropp.intellij.makefile, ru.trylogic.idea.gitlab.integration, mobi.hsz.idea.gitignore, com.kalessil.phpStorm.phpInspectionsEA, de.espend.idea.php.annotation, de.espend.idea.php.phpunit, ru.adelf.idea.dotenv
